src/sentry/static/sentry/app/views/performance/trends/chart.tsx

```diff
@@ -62,59 +62,127 @@ function getLegend() {
     align: 'left',
     textStyle: {
       verticalAlign: 'top',
-      fontSize: 11,
+      fontSize: 12,
       fontFamily: 'Rubik',
     },
     data: [
       {
-        name: 'Previous Period / This Period',
+        name: 'Baseline',
         icon:
           'path://M180 1000 l0 -40 200 0 200 0 0 40 0 40 -200 0 -200 0 0 -40z, M810 1000 l0 -40 200 0 200 0 0 40 0 40 -200 0 -200 0 0 -40zm, M1440 1000 l0 -40 200 0 200 0 0 40 0 40 -200 0 -200 0 0 -40z',
       },
+      {
+        name: 'Releases',
+        icon: 'line',
+      },
     ],
   };
   return legend;
 }
 
-function getIntervalLine(series: Series[], intervalRatio: number) {
-  if (!series.length || !series[0].data || !series[0].data.length) {
+function getIntervalLine(
+  series: Series[],
+  intervalRatio: number,
+  transaction?: NormalizedTrendsTransaction
+) {
+  if (!transaction || !series.length || !series[0].data || !series[0].data.length) {
     return [];
   }
-  const additionalLineSeries = [
-    {
-      color: theme.gray700,
-      data: [],
-      markLine: {
-        data: [] as any[],
-        label: {show: false},
-        lineStyle: {
-          normal: {
-            color: theme.gray700,
-            type: 'dashed',
-            width: 2,
-          },
-        },
-        symbol: ['none', 'none'],
-        tooltip: {
-          show: false,
+
+  const seriesStart = parseInt(series[0].data[0].name as string, 0);
+  const seriesEnd = parseInt(series[0].data.slice(-1)[0].name as string, 0);
+
+  if (seriesEnd < seriesStart) {
+    return [];
+  }
+
+  const periodLine = {
+    data: [] as any[],
+    color: theme.gray700,
+    markLine: {
+      data: [] as any[],
+      label: {} as any,
+      lineStyle: {
+        normal: {
+          color: theme.gray700,
+          type: 'dashed',
+          width: 1,
         },
       },
-      seriesName: 'Previous Period / This Period',
+      symbol: ['none', 'none'],
```

Comment:
There is a `MarkLine` class that has this as the default.

Reply:
merging for now, I'll switch this in a later PR